### ISSUE TITLE
Fingerprint with database

### ISSUE THREAD

**Issue by teggsdgreat** [mjwheatley/ionic2-meteor-boilerplate#5]
Good day, 
please is it posible to use and external database like MySQL with this code, if yes any advice on the head way.

Thanks

**Comment by mjwheatley** [comment]
Yes, it may be possible.  Please refer to this link: https://forums.meteor.com/t/how-to-use-ms-sql-server-in-meteor/33115/13

**Comment by teggsdgreat** [comment]
Thanks a lot i will chect it out
